CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.28.1]
+### Fixed
+- Fix bug where front-end error was thrown when setting `Graph.figure = {}` (fixes [#260]).
```

Comment:
Could you add a `### Changed` that mentions that "candlestick and OHLC charts are now plotted using the Plotly.react method instead of the Plotly.newPlot method."